plugin/pkg/shoot/validator/admission.go

```diff
@@ -267,8 +267,13 @@ func (v *ValidateShoot) Admit(a admission.Attributes, o admission.ObjectInterfac
 			oldShoot:     oldShoot,
 		}
 		allErrs field.ErrorList
+		maxPod  int32
```

Comment:
I don't see this getting used for anything. Can you simply check out the `plugin` dir to `master` again?

Reply:
maxPod is being calculated by looking at all the workers plus the global kubelet max pod settings.

For instance here: https://github.com/gardener/gardener/blob/cab503195d171c9ac72f4f1de36a333389b78bd9/plugin/pkg/shoot/validator/admission.go#L292

Reply:
I see this, but what is done with this value? Nothing, right? So why do you calculate this at all?

Reply:
My bad, this is a left over before refactoring. Removed it